ansible-deployment/switch/tasks/switches.j2

```diff
@@ -111,3 +111,16 @@ mac=0c:29:ef:bd:a8:a2
 manage_vlans=True
 ansible_connection=local
 stp_edge=True
+[ansible:OCT4-SW-ESIDEV]
+ansible_connection=ansible.netcommon.network_cli
+ansible_network_os=dellemc.os9.os9
+ansible_host=10.2.4.100
+ansible_user=admin
+ansible_ssh_pass={{ ansible_sw_moc_ssh_password }}
```

Comment:
I think you want to change these to `{{ ansible_switch_ssh_password }}`

Reply:
Yes, I've updated the PR